modules/fishing.py

```diff
@@ -30,29 +24,37 @@
 shark = emoji.emojize(':shark:')  # 98 - 100 100d
 
 
-class FishingSession:
-    id: int
-    state: int
+class FishingState(Enum):
+    WAITING_FOR_BITE = 1
+    FISH_BITING = 2
+    HOOKING = 3
 
 
-async def _bite_timer(ctx: Context, fisher: int):
+sessions: Dict[int, FishingState] = {}
```

Comment:
I think you were on the right track, but using a hashmap (called a dictionary `dict` in Python) instead of a `list` is much more efficient for lookup by key (in our case, author id).